wit-definitions/aggregator/wit/aggregator.wit

```diff
@@ -25,6 +25,7 @@ interface aggregator {
         workflow-id: workflow-id,
         envelope: envelope,
         signature: envelope-signature,
+        origin-tx-hash: list<u8>, // the tx hash from the original transaction that triggered this workflow (used for reorg handling)
```

Comment:
tx block too?

Reply:
Yes. Then it would be a complete reference and we could see if the block at that height still has teh same hash.